frontend/src/app/modules/bim/ifc_models/xeokit/xeokit-server.ts

```diff
@@ -0,0 +1,83 @@
+// @ts-ignore
+import {utils} from "@xeokit/xeokit-sdk/src/viewer/scene/utils";
+import {PathHelperService} from "../../../common/path-helper/path-helper.service";
+import {GonType} from "../../../common/gon/gon.service";
+import {IFCGonDefinition} from "../pages/viewer/ifc-models-data.service";
+
+interface Window {
+  gon:GonType;
+}
+
+/**
+ * Default server client which loads content via HTTP from the file system.
+ */
+export class XeokitServer {
+  private window:Window;
+  private ifcModels:IFCGonDefinition;
+  /**
+   *
+   * @param config
+   * @param.config.pathHelper instance of PathHelperService.
+   */
+  constructor(private pathHelper:PathHelperService) {
+    this.window = window;
```

Comment:
For the same typing reason, seems that we could get rid of the window variable and directly do this.ifcModels = window.gon.ifc_models;